packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.1.1]
+
+### Removed
+
+- Removed fee history fallback ([#4210](https://github.com/MetaMask/core/pull/4210))
```

Comment:
Should this be considered a breaking change? cc @matthewwalsh0 

The way it's described, it seems like it mainly improves reliability of our gas estimates. But it seems possible for this to lead to a degradation in some cases (e.g. for Ethereum-like networks that our API does not support)

Reply:
The clients already support estimates of alternate types meaning this should already be handled since the fallback that is still in use is unchanged, just more likely to occur now.

So in a technical and functional sense, I'd argue this isn't a breaking chance since there is no API changes nor any increased risk of errors.

Reply:
Fair enough! Makes sense.

I guess I was thinking that it could impact clients because it might result in worse estimates in some situations now. But it would be non-breaking even if that was true, because clients already need to handle the fallback which is unchanged here.